third_party_release

```diff
@@ -1,23 +1,10 @@
-#
-# MAINTAINER
-#
-# This file is used for the CMake build.
-#
-# When changing (add, upgrade, remove) dependencies
-# please update:
-# - the Bazel build, see file
-#   <root>/bazel/repository.bzl
-# - git submodule, see command
-#   git submodule status
-#
-
 gRPC=v1.48.1
 thrift=0.14.1
 abseil=20220623.1
 benchmark=v1.5.3
-googletest=release-1.10.0-459-ga6dfd3ac
+googletest=release-1.12.1
 ms-gsl=v3.1.0-67-g6f45293
 nlohmann-json=v3.10.5
-opentelemetry-proto=v0.19.0
+opentelemetry-proto=v0.18.0
```

Comment:
Why changing opentelemetry-proto back to 0.18.0 ?

Reply:
This is generated by `ci/do_ci.sh third_party.tags` , Maybe we should update the submodule instead.